### ISSUE TITLE
File tree only shows after opening the connect to remote project

### ISSUE THREAD

**Issue by rafaelgaspar** [facebook/nuclide#122]
Steps to reproduce, it was working fine on 0.0.21 as far as I can remember.

1. Open Atom, no File tree appears.
2. Execute "Nuclide File Tree: Open", nothing happens.
3. Execute "Nuclide Remote Projects: Connect" and close the dialog.
4. Execute "Nuclide File Tree: Open", the File tree appears.

My config.cson:
```
"*":
  "exception-reporting":
    userId: "bca551b3-40d0-cfda-fd80-262e4182bc5d"
  welcome:
    showOnStartup: false
  core:
    projectHome: "/Users/rafaelgaspar/Projects"
    disabledPackages: [
      "tree-view"
    ]
  editor:
    invisibles: {}
    showIndentGuide: true
    showInvisibles: true
    preferredLineLength: 120
    softWrap: true
    softWrapAtPreferredLineLength: true
  "file-watcher":
    promptWhenFileHasChangedOnDisk: false
  nuclide:
    remoteProjectsConfig: []
  "nuclide-flow": {}
  minimap: {}
  zentabs:
    neverCloseUnsaved: true
    neverCloseDirty: true
    neverCloseNew: true
  "linter-ruby":
    rubyExecutablePath: "/Users/rafaelgaspar/.rvm/bin/ruby"
  "linter-erb":
    erbExecutablePath: "/Users/rafaelgaspar/.rvm/bin/erb"
  linter: {}
  pigments: {}
```

**Comment by jssln** [comment]
Thanks for the report! We'll take a look. cc @oclbdk

**Comment by rafaelgaspar** [comment]
So far it seams to be fixed on 0.0.24.

**Comment by ssorallen** [comment]
@rafaelgaspar Let me know if you are still seeing this. 'nuclide-file-tree' went through a rewrite to better handle large remote directories, and so I suspect any issue was in the previous version of the code.
